t/03-testapi.t

```diff
@@ -118,6 +119,36 @@ my $mod2 = Test::MockModule->new('testapi');
 
 my $mock_bmwqemu = Test::MockModule->new('bmwqemu');
 
+subtest '_handle_found_needle' => sub {
+    my $mocked_needle = Test::MockModule->new('needle');
+    $mocked_needle->mock('new', sub { return bless({name => 'needlename'}) });
+    my $mocked_encode = Test::MockModule->new('Encode');
+    $mocked_encode->mock('decode_base64', sub { return 'A value' });
+    my $mocked_tinycv = Test::MockModule->new('tinycv');
+    $mocked_tinycv->mock('from_ppm', sub { return 'A Value' });
```

Comment:
```suggestion
    $mocked_tinycv->mock(from_ppm => 'A Value');
```
The same counts for further occurrences. We usually avoid quotes and you can just pass the scalar value as 2nd argument instead of a function returning it.

Reply:
Thanks. It is done.